examples/ex14.cpp

```diff
@@ -43,18 +43,33 @@ int main(int argc, char *argv[])
    const char *mesh_file = "../data/star.mesh";
    int ref_levels = -1;
    int order = 1;
+   bool rk = false;
+   int rk_num_points = 4;
+   double rbf_h = 2.01;
+   int rbf_type = 6;
+   double rk_face_factor = 0.0;
    double sigma = -1.0;
    double kappa = -1.0;
    double eta = 0.0;
-   bool visualization = 1;
+   bool visualization = true;
 
    OptionsParser args(argc, argv);
    args.AddOption(&mesh_file, "-m", "--mesh",
                   "Mesh file to use.");
    args.AddOption(&ref_levels, "-r", "--refine",
                   "Number of times to refine the mesh uniformly, -1 for auto.");
    args.AddOption(&order, "-o", "--order",
-                  "Finite element order (polynomial degree) >= 0.");
+                  "Finite element order (polynomial degree) >= 0 OR reproducing kernel order.");
+   args.AddOption(&rk, "-rk", "--rk", "-no-rk", "--no-rk",
+                  "Use reproducing kernel functions");
+   args.AddOption(&rk_num_points, "-rkp", "--rk-num-points",
+                  "Number of reproducing kernel points across each dimension of element");
+   args.AddOption(&rbf_h, "-rbfh", "--rbf-h",
```

Comment:
You've doubled the number of options in these examples, thus changing the focus. I'd recommend using default values (ones that work best) for the parameters, and keep the only the `-rk` switch.

Reply:
Done.